### ISSUE TITLE
`node_modules` resolution (~) only works for files placed in the root directory

### ISSUE THREAD

**Issue by ikatyang** [mrmlnc/vscode-scss#126]
`node_modules` resolution (~) only works if the file which uses `@import` is in the root directory (where the `node_modules` directory stands), the intellisense does not work if the file is placed in a sub-directory (e.g., `src/`), which means it does not work in the monorepo scenario as well where the `node_modules` directory could be hoisted.

Steps to Reproduce:

```sh
mkdir scss-intellisense-node-modules-bug
cd scss-intellisense-node-modules-bug
mkdir -p node_modules/my-package
echo '$my-variable: "something";' > node_modules/my-package/index.scss
mkdir src
echo '
@import "~my-package/index.scss";
$test: $;
' > src/index.scss

# 1. open scss-intellisense-node-modules-bug folder in vscode
# 2. open src/index.scss
# 3. trigger intellisense on the last $ character
# 4. $my-variable is not available
```

Environments:

- VS Code Version: 1.48.2
- SCSS IntelliSense Version: 0.9.0
- Operating System: macOS

Additional Context:
- it seems the `node_modules` resolution is hard-coded to use the `node_modules` directory that is directly next to the scss files. [`resolve-from`](https://github.com/sindresorhus/resolve-from) may be helpful.
  https://github.com/mrmlnc/vscode-scss/blob/593d65f1f422966d531d839fdf9dd94de1012dc6/src/utils/document.ts#L65-L74
- the temporary workaround I used for now is to place an "index" scss file in the root directory so that those scss files in the sub-directories can trigger the scss intellisense via the `(implicitly)` resolution.

**Comment by david-sharer** [comment]
Same issue, solved with the same workaround for @angular/material.


```scss
// file: /autocomplete-patch.scss
// (alongside node_modules)
@import 'node_modules/@angular/material/theming';
```

```scss
// file: /src/variables.scss
@import '../autocomplete-patch.scss';
```

inside /src/variables.scss
![image](https://user-images.githubusercontent.com/26589071/98308470-ee56be00-1f7c-11eb-873b-ac487d4ea002.png)

**Comment by mrmlnc** [comment]
Currently fixed in the `master` branch and will be shipped with `0.10.0`. Let me know if I'm wrong.

<img width="684" alt="Screenshot 2021-05-08 204519" src="https://user-images.githubusercontent.com/7034281/117548581-62624380-b03e-11eb-906c-36562ee159ed.png">
